rtl/eclrtl/eclrtl.hpp

```diff
@@ -74,6 +74,8 @@ interface IStrRegExprFindInstance
 {
     virtual bool found() const = 0;
     virtual void getMatchX(size32_t & outlen, char * & out, unsigned n = 0) const = 0;
+    virtual bool nextMatch() = 0;
```

Comment:
nextMatch() is used foldConstantOperator()  - ecl/hql/hqlfold.cpp lines 2190 and 2223.